src/main/java/com/oracle/nosql/spring/data/repository/support/NosqlEntityInformation.java

```diff
@@ -378,4 +389,8 @@ public void setTimeout(int milliseconds) {
         }
         timeout = milliseconds;
     }
+
+    public TimeToLive getTtl() {
```

Comment:
Add a javadoc comment with @since version.

Reply:
Added javadoc. Please review

Reply:
Please add a note saying that this is used only if/when the table is created by Spring SDK and it will not reflect the TTL of an already created table.

Reply:
Added.